packages/activemodel-adapter/lib/system/active-model-serializer.js

```diff
@@ -166,7 +166,10 @@ var ActiveModelSerializer = RESTSerializer.extend({
     if (Ember.isNone(belongsTo)) {
       json[jsonKey] = null;
     } else {
-      json[jsonKey] = capitalize(camelize(belongsTo.typeKey));
+      json[jsonKey] = belongsTo.typeKey.split("/").map(function(part) {
```

Comment:
We can't use map here due to IE8 support (I'm not sure when we're dropping it). You can use Ember.EnumerableUtils.map instead.